package.json

```diff
@@ -11,11 +11,15 @@
     "package_update": "git submodule update --remote --recursive && npm install"
   },
   "dependencies": {
+    "@heroicons/react": "^2.1.4",
```

Comment:
those are able to import from shared-components

Reply:
can you please explain this comment

Reply:
Since I imported a shared package that included most of the deps and that will auto link from that package, so when you run `npm i` the package you added to the root project is not necessary to add these to stand out the deps